File: php/Car.php
<?php
class Car {

    public $id = integer;
    public $license = string;
    public $driver = string;
    public $passengers = integer;
    
    public function __construct($license, $driver) {
        $this->license = $license;
        $this->driver = $driver;
    }

    public function PrintDataCar(){
        echo "\nLicense: $this->license, Conductor: {$this->driver->name}, Document: {$this->driver->document}\n";
    }
}

File: java/Car.java
class Car {
   
    Integer id;
    String license;
    Account driver;
    Integer passenger;
    
    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
    }

    void printDataCar(){
        System.out.println("\nLicense: " + license + " \nDriver: " + driver.name + "\n");
    }

}
